ledger/ledger.go

```diff
@@ -443,6 +443,13 @@ func (l *Ledger) GetCreator(cidx basics.CreatableIndex, ctype basics.CreatableTy
 	return l.accts.GetCreatorForRound(l.blockQ.latest(), cidx, ctype)
 }
 
+// GetAccountDeltasForRound retrieves the AccountDeltas object from the accountUpdates cache
+func (l *Ledger) GetAccountDeltasForRound(rnd basics.Round) (ledgercore.AccountDeltas, error) {
```

Comment:
I guess we'll have an equivalent `GetBoxDeltasForRound` soon?

Reply:
Yes, my expectation is that the ledger should expose all of the deltas we'll need to return via the API, and the handlers will be responsible for fetching those from the ledger and combining them into a single `RoundDeltas` response object.

Reply:
To expand on the idea above, I wonder if we can simplify the accountUpdates tracker a bit by getting rid of the three different `deltas`, `creatableDeltas`, and `kvDeltas`. Instead, just have `deltas`, but have it hold onto `ledgercore.stateDelta`.  Then you have a simple implementation of what I described above. (And yes, it means we're holding on a bit more than we need in `deltas`, but nowadays we only hold onto a few rounds in there, so simplicity is better.